src/App/rootReducer.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import wishlistView from '../WishlistView/reducer';
```

Comment:
wishlistView could be named more aptly. What is a "view"? Call it `wishlist` or give it a specific name based on its purpose.